packages/m/mariadb-connector-c/xmake.lua

```diff
@@ -0,0 +1,107 @@
+package("mariadb-connector-c")
+    set_homepage("https://github.com/mariadb-corporation/mariadb-connector-c")
+    set_description("MariaDB Connector/C is used to connect applications developed in C/C++ to MariaDB and MySQL databases.")
+    set_license("LGPL-2.1")
+    add_urls("https://github.com/mariadb-corporation/mariadb-connector-c/archive/refs/tags/v$(version).tar.gz")
+    add_versions("3.1.13", "361136e9c365259397190109d50f8b6a65c628177792273b4acdb6978942b5e7")
+    add_deps("cmake")
+
+    add_links("mariadb")
+    add_linkdirs("lib/mariadb/")
+
+    -- Windows specific options
```

Comment:
Remove unnecessary comment

Reply:
How is that unnecessary? I feel like it's important to note platform-specific options.

Reply:
The is_plat("windows") statement can basically explain these